airflow/config_templates/config.yml

```diff
@@ -335,6 +335,15 @@
       type: string
       example: ~
       default: "True"
+    - name: store_dag_code
```

Comment:
Let's move this to live next to store_serialized_dags as these two settings are related.

Reply:
I have moved it just below min_serialized_dag_update_interval because it seemed to me that it should stay just below store_serialized_dags. But I have no strong opinion here.